packages/sanity/src/core/form/inputs/files/FileInput/FileActionsMenu.tsx

```diff
@@ -87,6 +88,8 @@ export function FileActionsMenu(props: Props) {
     }
   }, [isMenuOpen, menuElement])
 
+  const {t} = useTranslation('sanity')
```

Comment:
The default namespace is `studio`, not `sanity` - and can be left out when it is the default
```suggestion
  const {t} = useTranslation()
```

Reply:
ok, I copied that from somewhere else. I'll update that one as well.